cloudify_tg/tasks.py

```diff
@@ -101,3 +103,48 @@ def run_command(command, tg, options=None, force=False, *_, **__):
 
     tg.update_command_options({command: options})
     tg.execute(command)
+
+
+@operation
+@decorators.skip_if_existing
+def install(ctx, **_):
+    # folder
+    installation_dir = utils.get_node_instance_dir()
+    # The path to the terragrunt binary executable.
+    executable_path = utils.get_executable_path()
+    # Location to download the Terraform executable binary from.
+    installation_source = utils.get_installation_source()
+
+    if os.path.isfile(executable_path):
+        ctx.logger.info(
+            'Terragrunt executable already found at {path}; '
+            'skipping installation of executable'.format(
+                path=executable_path))
+    else:
+        ctx.logger.warn('You are requesting to write a new file to {loc}. '
+                        'If you do not have sufficient permissions, that '
+                        'installation will fail.'.format(
+                            loc=executable_path))
+        utils.install_binary(
+            installation_dir, executable_path, installation_source)
```

Comment:
Here you could add the file name:
```suggestion
            binary_name = "terragrunt" # or "terraform" or whatever.
            os.path.join(installation_dir, binary_name), executable_path, installation_source)
```